web/src/scss/app.scss

```diff
@@ -0,0 +1,186 @@
+/*
+ * Globals
+ */
+
+body {
+  font-family: Georgia, "Times New Roman", Times, serif;
+  color: #555;
+}
+
+h1, .h1,
+h2, .h2,
+h3, .h3,
+h4, .h4,
+h5, .h5,
+h6, .h6 {
+  margin-top: 0;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: normal;
+  color: #333;
+}
+
+/*
+ * Override Bootstrap's default container.
+ */
+ 
+ .container {
+  width: 1280px;
+}
+
+.leaderboard{
+  width: 900px;
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.header {
+  padding-top: 0px;
+  padding-bottom: 20px;
+}
+.title {
+  margin-top: 30px;
+  margin-bottom: 0;
+  font-size: 60px;
+  font-weight: normal;
+}
+
+/*
+ * Main column and sidebar layout
+ */
+
+.main {
+  font-size: 18px;
+  line-height: 1.5;
+}
+
+/* Sidebar modules for boxing content */
+.sidebar-module {
+  padding: 10px;
+  margin: 0 0px 0px;
+}
+.sidebar-module-inset {
+  padding: 0px;
+  background-color: #4a4a4a;
+  border-radius: 4px;
+}
+.sidebar-module-inset p:last-child,
```

Comment:
Since we are using sass http://sass-lang.com/, you could simplify that with
```sass
.sidebar-module-inset {
  p:last-child,
  ul:last-child,
  ol:last-child {
    margin-bottom: 0;
  }
}
```

If you don't like this syntax, don't use it. But considere reading http://sass-lang.com/guide.